lib/vanagon/platform/windows.rb

```diff
@@ -8,7 +8,12 @@ class Windows < Vanagon::Platform
       def generate_package(project)
         target_dir = project.repo ? output_dir(project.repo) : output_dir
         ["mkdir -p output/#{target_dir}",
-        "touch output/#{target_dir}/#{project.name}-#{project.version}-#{project.release}-#{@architecture}.msi"]
+        "mkdir -p #{project.name}/tools",
```

Comment:
It already is! All of this gets run, by default, according to vanagon, under `/var/tmp/tmp.#{somerandomstring}`